sycl/include/sycl/backend_types.hpp

```diff
@@ -56,7 +56,7 @@ inline std::ostream &operator<<(std::ostream &Out, backend be) {
   case backend::ext_oneapi_hip:
     Out << "ext_oneapi_hip";
     break;
-  case backend::ext_native_cpu:
+  case backend::ext_oneapi_native_cpu:
     Out << "ext_native_cpu";
```

Comment:
This needs to be changed as well. It will modify `sycl-ls` output but that would be a separate bugfix, not necessarily made in this area of the code.

Reply:
Thanks for pointing this out, I've made the change, I don't think the output of `sycl-ls` is used anywhere right now for native cpu